jax/_src/nn/initializers.py

```diff
@@ -37,13 +38,15 @@
 DTypeLikeInexact = Any  # DTypeLikeFloat | DTypeLikeComplex
 RealNumeric = Any  # Scalar jnp array or float
 
+@set_module('jax.nn.initializers')
```

Comment:
I'm sort of tempted to write
```
export = set_module('jax.nn.initializers')
```
above somewhere and not repeat the name. Up to you.

Reply:
Good idea - Done